### ISSUE TITLE
Can't watch this specific video(Link inside). Apparently it's need a disclaimer or somethings similar....

### ISSUE THREAD

**Issue by DjDiabolik** [anxdpanic/plugin.video.youtube#289]
### Context

Please provide any relevant information about your setup

* Add-on Version: Current... 6.8.18 Matrix 1 - Unofficial version
* Kodi Version: 19.4
* Kodi GUI Language: Italian
* Operating System: OSMC current build on a Raspberry Pi2
* Operating System Language: thinks english ??

------

### Expected Behavior

I have apparently all setted correctly.... i have no issue whit "My Subscription" list or somethigs similar. I have also applyed my APi whit all need but i have this issue whit only this specific video:
https://www.youtube.com/watch?v=AwhG7TwreJo

If i open this link on a PC Browser i have to accept a disclaimer... it's appears somethigs similar message:
The following content may include topics related to suicide or self-harm.

I can click on BIG grey button texted on it "I understand and wish to proceed" and only after that i can watch this video

------

### Current Behavior

About 5 minutes i have tryed to open this same video on kodi........... the addons goes in errors and appears a notification un upper right corner of my tv.. it's say somethigs about i need to confirm my age after accessing to my principal account.
On kodi.log it's appears this when try to reproduce this video:
https://justpaste.it/2yhiz
Or snippet code can read here:
https://pastecode.io/s/xswks7cf

I don't know if it's a bug....... i hope i have open a good issue to made some enachment to current addons.

**Comment by DjDiabolik** [comment]
Noone as some info about this issue ?

**Comment by chriss158** [comment]
Same issue for me.
Video: https://www.youtube.com/watch?v=WzPcxe32-HY

**Comment by DjDiabolik** [comment]
Yeah @chriss158 it's same issue. Inappropriate suicide and other "stupid" idea :)
Youtube need to stop to made auto-decision :)

**Comment by FunctionalHacker** [comment]
This is a duplicate of #278. Please close this one
